solr/contrib/ltr/src/test-files/solr/collection1/conf/schema.xml

```diff
@@ -38,6 +51,11 @@
   <copyField source="title" dest="text"/>
   <copyField source="description" dest="text"/>
 
+  <copyField source="popularity" dest="dvIntPopularity"/>
+  <copyField source="popularity" dest="dvLongPopularity"/>
+  <copyField source="popularity" dest="dvFloatPopularity"/>
+  <copyField source="popularity" dest="dvDoublePopularity"/>
```

Comment:
observation: `copyField` here works for the `dv*Popularity` fields but the `dv(Int|Long|Float|Double)Field` fields cannot use it since the defaulting logic and the popularity/10 logic is happening there.